compiler/src/dmd/statementsem.d

```diff
@@ -2285,6 +2285,11 @@ package (dmd) extern (C++) final class StatementSemanticVisitor : Visitor
             {
                 s = new BreakStatement(ss.loc, null);   // default for C is `default: break;`
             }
+            else if (sc.flags & (SCOPE.ctfe | SCOPE.compile | SCOPE.ctfeBlock))
```

Comment:
I'm seeing this grouping of flags crop up a lot across a number PRs, would be better to put in their own named constant or function.

Reply:
I plan on saving refactorings for later.

Reply:
In order to minimize LOC changes for substantive changes.